vspace/src/main/java/edu/asu/diging/vspace/core/model/SortByField.java

```diff
@@ -0,0 +1,16 @@
+package edu.asu.diging.vspace.core.model;
+
+public enum SortByField {
+    CREATION_DATE("creationDate");
+
+    private final String value;
+
+    private SortByField(String value) {
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
```

Comment:
I wouldn't do this. This might have unintended side effects if used in a jsp page. Just add a method to retrieve the value.